dash_bio/metadata.json

```diff
@@ -1684,30 +1700,22 @@
         "required": false,
         "description": "Dash callback that returns True if rotated, and false if not."
       },
-      "resolution": {
-        "type": {
-          "name": "number"
-        },
-        "required": false,
-        "description": "The resolution of cytogenetic bands to show for each chromosome.\nThe quantity refers to approximate value in bands per haploid set (bphs).\nOne of 450, 550, or 850."
-      },
-      "rows": {
-        "type": {
-          "name": "number"
-        },
-        "required": false,
-        "description": "Useful for putting ideogram into a small container,\nor when dealing with genomes that have many chromosomes.\nNote: Not fully working needs to be fixed by developer.",
-        "defaultValue": {
-          "value": "1",
-          "computed": false
-        }
-      },
       "sex": {
         "type": {
-          "name": "string"
+          "name": "enum",
+          "value": [
+            {
+              "value": "'male'",
+              "computed": false
+            },
+            {
+              "value": "'female'",
+              "computed": false
+            }
+          ]
         },
         "required": false,
-        "description": "Useful for omitting chromosome Y in female mammals.\nCurrently only supported for organisms that use XY sex-determination.\nAll string values will show chromosome Y, only the value 'female' will hide it"
+        "description": "Useful for omitting chromosome Y in female mammals.\nCurrently only supported for organisms that use XY sex-determination."
```

Comment:
For example, here, I think "species" would be more fitting than "organisms." What do you think @NadiaTahiri ?